lib/material/material_gui.flow

```diff
@@ -372,6 +372,16 @@ export {
 	// Identical to MFrame, only the frame is placed on top of the "content"
 	MOverlay(margin : double, radius : double, style : [MGraphicsStyle], content : Material) -> Material;
 	MOverlayT(margin : Transform<double>, radius : Transform<double>, style : Transform<[MGraphicsStyle]>, content : Material) -> Material;
+
+	// Helper for Segmented buttons 
+	// https://m3.material.io/components/segmented-buttons/overview
+	MSegmentedButtons(items : [SegmentButtonItem], styles : [SegmentButtonsStyle]) -> Material;
+
+	SegmentButtonItem(text : Maybe<MText>, icon : Maybe<MIcon>, state : [MButtonState], selected : DynamicBehaviour<bool>);
```

Comment:
text : Maybe<string>, icon : Maybe<string>? Do we really need to provide styles? Still fine though.